[Observability] 9.4.0 release notes (#6077)
elastic/docs-content: release-notes/elastic-observability/index.md

* Renames workflow label references to tags throughout the Observability UI for consistency [#260329]({{kib-pull}}260329).
* Fixes crashes caused by malformed URLs in plugins; affected instances now attempt automatic recovery [#257245]({{kib-pull}}257245).
* Fixes dashboard scanning to correctly include sections [#254600]({{kib-pull}}254600).
* Fixes the annotation API on Elastic Serverless [#254285]({{kib-pull}}254285).
* Fixes a race condition in the data quality controller that caused incorrect Discover navigation [#254139]({{kib-pull}}254139).
* Fixes an infinite loading loop in the document flyout when using relative time ranges [#251647]({{kib-pull}}251647).
* Adjusts the panel height in Discover for improved layout [#250778]({{kib-pull}}250778).
* Fixes **View In Context** links in the Discover modal that were not respecting the selected date range [#248939]({{kib-pull}}248939).
* Fixes loss of UI state in signal-specific Discover flyout tabs when refreshing a query [#248203]({{kib-pull}}248203).
* Fixes broken breadcrumbs and sidebar navigation for Data Visualizer and AIOps within solution views [#248167]({{kib-pull}}248167).

> * Fixes double scrollbars appearing in fullscreen flyouts [#247744]({{kib-pull}}247744).

Reviewer: ```suggestion
* Fixes double scrollbars appearing in fullscreen flyouts [#247744]({{kib-pull}}247744).
* Fixes links to Stack Management and Observability rules that routed to the wrong destination, causing the navigation menu to disappear [#262903]({{kib-pull}}262903).
```

% ### Fixes [elastic-observability-next-fixes]
% *

## 9.4.0 [elastic-observability-9.4.0-release-notes]

### Features and enhancements [elastic-observability-9.4.0-features-enhancements]

* Removes the confirmation modal when switching to AI Agent mode [#264839]({{kib-pull}}264839).
* Adds the `observability.investigation` skill to the Elastic Agent, providing structured investigation methodology across all Observability tools for service health, dependency analysis, and log and trace correlation [#262293]({{kib-pull}}262293).

> * Adds AI Agent as the default Observability chat experience; users can return to the classic AI Assistant through **GenAI Settings** [#260570]({{kib-pull}}260570).

Author: ```suggestion
* Adds AI Agent as the default Observability chat experience. AI Assistant has been deprecated. Users can return to AI Assistant through **GenAI Settings** [#260570]({{kib-pull}}260570).
```

* Fixes crashes caused by malformed URLs in plugins; affected instances now attempt automatic recovery [#257245]({{kib-pull}}257245).
* Fixes dashboard scanning to correctly include sections [#254600]({{kib-pull}}254600).
* Fixes the annotation API on Elastic Serverless [#254285]({{kib-pull}}254285).
* Fixes a race condition in the data quality controller that caused incorrect Discover navigation [#254139]({{kib-pull}}254139).
* Fixes an infinite loading loop in the document flyout when using relative time ranges [#251647]({{kib-pull}}251647).
* Adjusts the panel height in Discover for improved layout [#250778]({{kib-pull}}250778).
* Fixes **View In Context** links in the Discover modal that were not respecting the selected date range [#248939]({{kib-pull}}248939).
* Fixes loss of UI state in signal-specific Discover flyout tabs when refreshing a query [#248203]({{kib-pull}}248203).
* Fixes broken breadcrumbs and sidebar navigation for Data Visualizer and AIOps within solution views [#248167]({{kib-pull}}248167).
* Fixes double scrollbars appearing in fullscreen flyouts [#247744]({{kib-pull}}247744).

> * Fixes links to Stack Management and Observability rules that routed to the wrong destination, causing the navigation menu to disappear [#262903]({{kib-pull}}262903).

Reviewer: Don't need to release-note this. The bug was introduced and fixed during the 9.4 cycle so it was never user-facing.

```suggestion
```
